stable/graphite/templates/statefulset.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: apps/v1beta1
+kind: StatefulSet
```

Comment:
Currently not sure if we should make this configurable to be backwards compatible.
Thougts?

Reply:
Disagree - Graphite's model of meaningful named agents in a cluster makes a StatefulSet the right choice.